### PR TITLE
Fix AP handling and remove ctrl_interface as not needed for ms2.0

### PR DESCRIPTION
**Pull request by joenpera**
Jira-Id: SECO-4998

### CODE REVIEW

File: modules/sc-mesh-secure-deployment/src/nats/initd/S90APoint [hunk 1]
```diff
@@ -53,21 +53,19 @@ case "$1" in
...
-
     stop)
         if "$daemon" --running --name "$name" --pidfiles "$pidfiles"
         then
             echo "Stopping $name..."
             "$daemon" --stop --name "$name" --pidfiles "$pidfiles"
             echo done.
 
-            NEW_PID=$(ps ax | grep -E "[h]ostapd-$RADIO_INDEX" | awk {'print $1'})
+            NEW_PID=$(ps ax | grep -E "[h]ostapd-$RADIO_INDEX" | awk "{'print $1'}")
```
Comment (pentestiing) [inline modules/sc-mesh-secure-deployment/src/nats/initd/S90APoint:63]: The `awk` command will fail, replace it to: `awk '{print $1}'`

Or even join it with the `grep`: `ps ax | awk "/ostapd-$RADIO_INDEX/{print \$1}"`

:bagel:

**Review by pentestiing — CHANGES_REQUESTED** [review]
:call_me_hand:

File: modules/sc-mesh-secure-deployment/src/nats/initd/S90APoint (hunk 1 above)
Comment (joenpera) [reply modules/sc-mesh-secure-deployment/src/nats/initd/S90APoint:63]: Yes, this PR is draft and under testing currently and not including the final changes.

**Review by saauvine — APPROVED** [review]

**Review by pentestiing — APPROVED** [review]
:signal_strength:

**Pull request merged**